addons/auth_signup/data/ir_config_parameter_data.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="1">
```

Comment:
I highly suspect this noupdate to be non-working in case of the function tag.

Reply:
I think this makes the function to be executed only once (on installation), but no more.

Reply:
this survive an `-u auth_signup` but not a `-i auth_signup`